beacon-chain/entrypoint.sh

```diff
@@ -1,9 +1,9 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
any difference between `bash` and `sh` here?

Reply:
Bash provides extended features that are not needed here